app/models/members/scopes/not_locked.rb

```diff
@@ -35,7 +35,7 @@ module NotLocked
       def not_locked
         includes(:principal)
           .references(:principals)
-          .merge(Principal.not_locked, rewhere: true)
+          .merge(Principal.not_locked)
```

Comment:
What was `rewhere: true` doing before? Couldn't find any good answers on this around

Reply:
This post explains it https://www.fastruby.io/blog/rails/upgrades/rails-merge-deprecation.html .
In this case using `.merge(..., rewhere: true)` makes sure that if we had a field condition on the original query and on the merge query, then the original field query would be overwritten by the merge query.

For example doing a merge with having `published: true` on the base query and having `published: false` on the merge query: 
 - Without using `rewhere` the query would be  "published = TRUE AND published = FALSE".
 - With `rewhere` the query is "published = FALSE".
 
Now in rails 7.1, the latter behaviour became the default, so using rewhere is deprecated.